load-test/multi_msg_load.sh

```diff
@@ -0,0 +1,73 @@
+#/bin/sh
+
```

Comment:
please add a documentation to this script. What are the parameters? Also we need a help message.